silverstripe/silverstripe-sqlite3
# MNT Add changes for compatibility with CMS5 and testing with framework

### Description
- Replace old method `first` to new method `record`;
- Remove `__config.yml` from root folder and add empty `config.yml` in `_config` folder;
- Add jobs to GA to test together with `framework`;
- Add test cases for this module;
- Remove `seek` method as deprecated and removed from framework ORM `Query::class`
`Query now implements IteratorAggregate instead of Iterator. This means seek() and other iterator methods are no longer available on this class and its subclasses. Use getIterator() instead.` [See changelog](https://docs.silverstripe.org/en/5/changelogs/5.0.0/#api-orm)

### Parent issue
- https://github.com/silverstripe/silverstripe-sqlite3/issues/68
